composer.json

```diff
@@ -20,33 +20,33 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": "^7.2 || ^8.0",
         "ext-json": "*",
         "barryvdh/reflection-docblock": "^2.0.6",
         "composer/composer": "^1.6 || ^2",
-        "doctrine/dbal": "~2.3",
-        "illuminate/console": "^6 || ^7 || ^8",
-        "illuminate/filesystem": "^6 || ^7 || ^8",
-        "illuminate/support": "^6 || ^7 || ^8",
+        "doctrine/dbal": "^2.6 || ^3",
+        "illuminate/console": "^8",
+        "illuminate/filesystem": "^8",
+        "illuminate/support": "^8",
         "phpdocumentor/type-resolver": "^1.1.0"
     },
     "require-dev": {
         "ext-pdo_sqlite": "*",
         "friendsofphp/php-cs-fixer": "^2",
-        "illuminate/config": "^6 || ^7 || ^8",
-        "illuminate/view": "^6 || ^7 || ^8",
-        "mockery/mockery": "^1.3.3",
-        "orchestra/testbench": "^4 || ^5 || ^6",
+        "illuminate/config": "^8",
+        "illuminate/view": "^8",
+        "mockery/mockery": "^1.4",
+        "orchestra/testbench": "^6",
         "phpunit/phpunit": "^8.5 || ^9",
-        "spatie/phpunit-snapshot-assertions": "^1.4 || ^2.2 || ^3 || ^4",
+        "spatie/phpunit-snapshot-assertions": "^3 || ^4",
         "vimeo/psalm": "^3.12"
```

Comment:
"vimeo/psalm": "^3.12" => `^3.12 || ^4.0`

Reply:
I'll postpone such changes, it's not focus of this PR.

Rather, I prefer to get rid of it from require-dev, as it doesn't need to be a source dependency, it's just tooling.

Reply:
Ok

Reply:
Actually, I already forgot I made a PR to bump it to ^4 => https://github.com/barryvdh/laravel-ide-helper/pull/1115

Btw, no idea why we would support older versions, don't see much gain? Feedback in https://github.com/barryvdh/laravel-ide-helper/pull/1115 please :)

Reply:
Thanks.